### ISSUE TITLE
Unable to determine button color contrast

### ISSUE THREAD

**Issue by henryjr1** [henryjr1/deeplyrooted_msu#50]
Button color contrast could not be determined on homepage. Investigate further.

**Comment by dcw293** [comment]
Home page does not have a button. Clarify the problem.

**Comment by ajm712** [comment]
I believe that I fixed the issue. It was the menu button
